src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/components/CourseProductCourseRuns/index.spec.tsx

```diff
@@ -147,11 +149,13 @@ describe('CourseProductCourseRuns', () => {
     it('renders a list of course runs with a call to action to enroll', async () => {
       const course: CourseLight = CourseLightFactory().one();
       const courseRuns: CourseRun[] = CourseRunFactory().many(2);
-      const order: OrderLite = CredentialOrderFactory().one();
+      const order = CredentialOrderFactory().one();
+      const product = ProductFactory().one();
+      product.contract_definition = undefined;
```

Comment:
note: i feel like it's a mistake in `factory.CredentialProductFactory`.
Most of the time, product will not have contract, default should be : `contract_definition: undefined`

```suggestion
     const product = ProductFactory({
       contract_definition: undefined
     }).one();
```